### PR TITLE
add aqua and remove a function: api change

### PR DESCRIPTION
**Pull request by ocots**
I have removed the function 

```julia
Base.:(\)(x::Description, y::Description)::Description = Tuple(setdiff(x, y))
```

since it is a type piracy. It is replaced by `remove(x, y)`.

Not very important but it changes the api and I use it in `OptimalControl`. So I have modified the Project.toml to go the release `0.13`.

### CODE REVIEW

**Comment by ocots** [comment]
Actually I don't use it anymore dans `OptimalControl` so it won't generate any problem but to be sure, `v0.13`.

File: test/test_aqua.jl
Comment (jbcaillau) [inline test/test_aqua.jl:1]: @ocots any (other) lesson from Aqua.jl test?

**Review by jbcaillau — APPROVED** [review]

File: test/test_aqua.jl
Comment (ocots) [reply test/test_aqua.jl:1]: Nope.

Comment (ocots) [reply test/test_aqua.jl:1]: We can merge later waiting for more changes.

**Comment by ocots** [comment]
@jbcaillau We should make the `v0.12.5` release before merging this PR.

**Pull request merged**